### PR TITLE
docs: update decline application

### PR DESCRIPTION
**Pull request by danakoch23**
## Description

Updated and improved documentation and description + added FAQs

## Why

More detailed and understandable

## Issue

n/a

## Checklist

- [x] I have followed the [contributing guidelines](https://github.com/eclipse-tractusx/portal-assets/blob/main/docs/developer/Technical%20Documentation/Dev%20Process/How%20to%20contribute.md#commit-and-pr-guidelines)

### CODE REVIEW

File: docs/user/01. Onboarding/03. Registration Approval/04. Decline Application.md [hunk 1]
```diff
@@ -1,10 +1,18 @@
...
+
+Here's what to expect after canceling:
+
+• The application status will be updated to reflect the cancellation. (This might take a short moment.)
+
+• You won't receive any further updates about the application.
 
 <br>
-<img width="827" alt="image" src="https://raw.githubusercontent.com/eclipse-tractusx/portal-assets/main/docs/static/application-requests-cancel-popup.png">
+<img src="https://github.com/danakoch23/portal-assets/blob/docs/onboardingupdate/docs/static/Decline%20Company%20Application2.png">
```
Comment (oyo) [inline docs/user/01. Onboarding/03. Registration Approval/04. Decline Application.md:15]: We leave fork name 'danakoch23' for now and replace with 'eclipse-tractusx' in a separate step.

**Review by oyo — APPROVED** [review]

**Review by oyo — APPROVED** [review]
lgtm :)

File: docs/user/01. Onboarding/03. Registration Approval/04. Decline Application.md [hunk 2]
```diff
@@ -13,6 +21,49 @@ To decline the application the "Cancel Application" Button is enabled for all ap
...
+including your spam folder, for any notifications regarding the cancellation.
+
+#### 3. How will I know if my application has been cancelled?
+
+You will receive an email notification if your application is canceled. Be sure to check your spam folder if you do not see the email in your inbox. Additionally, you can log in to your account to check the status of
+your application.
+
+#### 4. What should I do if my application was canceled by mistake?
+
+If your application was canceled by mistake, you will need to submit a new application. Currently, there is no way to reverse a cancellation.
```
Comment (jjeroch) [inline docs/user/01. Onboarding/03. Registration Approval/04. Decline Application.md:43]: Please add the Info: "A new registration invite needs to get triggered by the operator and the registration form needs to get newly filled out."

File: docs/user/01. Onboarding/03. Registration Approval/04. Decline Application.md [hunk 3]
```diff
@@ -13,6 +21,49 @@ To decline the application the "Cancel Application" Button is enabled for all ap
...
+
+If an operator cancels your application while it is being processed, the status will be updated to reflect the cancellation. You will no longer receive any updates about the application. Please check your email,
+including your spam folder, for any notifications regarding the cancellation.
+
+#### 3. How will I know if my application has been cancelled?
+
+You will receive an email notification if your application is canceled. Be sure to check your spam folder if you do not see the email in your inbox. Additionally, you can log in to your account to check the status of
+your application.
+
+#### 4. What should I do if my application was canceled by mistake?
```
Comment (jjeroch) [inline docs/user/01. Onboarding/03. Registration Approval/04. Decline Application.md:41]: Delete the words "by mistake"....from a customer point of view there can be many reasons why the operator cancelled it. Wrong data, mistake, etc. No matter what it was the answer below applies

**Review by jjeroch — CHANGES_REQUESTED** [review]

**Review by oyo — APPROVED** [review]

**Pull request merged**